functionary/ui/templates/partials/modal.html

```diff
@@ -0,0 +1,26 @@
+<div class="modal is-active" id="form-modal">
+    <div class="modal-background"></div>
+    <div class="modal-card">
+        <header class="modal-card-head">
+            <h1 class="modal-card-title is-2">
+                {% block modal_title %}{% endblock %}
+            </h1>
+            <button class="delete"
+                    aria-label="close"
+                    onclick="return this.parentNode.parentNode.parentNode.remove();"></button>
+        </header>
+        {% block form_declaration %}{% endblock %}
+        <section class="modal-card-body">
+            <div class="subtitle is-6">
+                {% block modal_description %}{% endblock %}
```

Comment:
Just so I understand, you don't want to include a description section for the modal?

The reason I decided to included a description was to allow us to give the user more context as to what the form would do upon submission.

Reply:
I guess I was thinking that the modal didn't necessarily have to be a form and instead just takes a generic body.  We don't have any instances now, but it's easily conceivable that you might want to have an informational modal somewhere that isn't really form.  In that situation, you could just have the text of the modal be the "description", but that felt a little weird to me.

I do think the footer situation ties our hands a little bit though, since the form can't be properly self-contained within the modal body.  So probably just keep it as is for now, we'll treat this modal as a "form modal" and maybe if / when we switch css libraries we'll be able to refactor it to be a little more generic.